modules/sovrnBidAdapter.js

```diff
@@ -172,6 +173,10 @@ export const spec = {
         deepSetValue(sovrnBidReq, 'source.tid', tid)
       }
 
+      if (config.getConfig('coppa') === true) {
```

Comment:
Can you get this from the request instead of config please?

Reply:
@ChrisHuie I could not find example of getting it from request unfortunately, all the adapters I saw do it in similar way. Could you point me to right direction?

Reply:
@ChrisHuie 
I've updated it. You can take a look